hedera-node/hedera-app/src/main/java/com/hedera/node/app/SessionContext.java

```diff
@@ -16,41 +16,5 @@
 
 package com.hedera.node.app;
 
-import static java.util.Objects.requireNonNull;
-
-import com.google.protobuf.Parser;
-import com.hederahashgraph.api.proto.java.Query;
-import com.hederahashgraph.api.proto.java.SignedTransaction;
-import com.hederahashgraph.api.proto.java.Transaction;
-import com.hederahashgraph.api.proto.java.TransactionBody;
-import edu.umd.cs.findbugs.annotations.NonNull;
-
-/**
- * This record keeps a list of everything that is used per-thread
- *
- * @param queryParser a parser for {@link Query}
- * @param txParser a parser for {@link Transaction}
- * @param signedParser a parser for {@link SignedTransaction}
- * @param txBodyParser a parser for {@link TransactionBody}
- */
-public record SessionContext(
-        @NonNull Parser<Query> queryParser,
-        @NonNull Parser<Transaction> txParser,
-        @NonNull Parser<SignedTransaction> signedParser,
-        @NonNull Parser<TransactionBody> txBodyParser) {
-
-    /**
-     * Constructor of {@code SessionContext}
-     *
-     * @param queryParser the {@link Query}-parser
-     * @param txParser the {@link Transaction}-parser
-     * @param signedParser the {@link SignedTransaction}-parser
-     * @param txBodyParser the {@link TransactionBody}-parser
-     */
-    public SessionContext {
-        requireNonNull(queryParser);
-        requireNonNull(txParser);
-        requireNonNull(signedParser);
-        requireNonNull(txBodyParser);
-    }
-}
+/** This record keeps a list of everything that is used per-thread */
+public record SessionContext() {}
```

Comment:
We think this can be removed, @netopyr has another branch where it gets removed. So we will look into this in the upcoming weeks after this gets merged.